code-security/admin_guide/get-started/connect-your-repositories/add-intellij.adoc

```diff
@@ -0,0 +1,53 @@
+:topic_type: task
+
+[.task]
+== Add IntelliJ to Prisma Cloud Code Security
+
+Integrating IntelliJ with Prisma Cloud Code Security makes it possible for you to identify misconfigurations before you commit your code, and avoid pull requests that can potentially fail builds due to undetected misconfigurations. Using Checkov, a code analysis tool that scans Infrastructure-as-code (IaC) files from frameworks such as Terraform, CloudFormation, and Azure Resource Manager (ARM) on IntelliJ gives you immediate detection of misconfigurations and inline code fixes.
```

Comment:
Is that the issue?
or is to help "get real-time scan results to help developers in the initial stages of development and quickly resolve issues early in the software lifecycle"

Reply:
The way I look at it absolve potencial crisis via "avoid pull requests that can potentially fail builds due to undetected misconfiguration" and with additional feature help of "immediate detection of misconfigurations and inline code fixes." Though this line summarizes everything "get real-time scan results to help developers in the initial stages of development and quickly resolve issues early in the software lifecycle"
I will rephrase this?